core/invoker/src/main/scala/whisk/core/containerpool/ContainerProxy.scala

```diff
@@ -107,6 +106,7 @@ class ContainerProxy(
   implicit val ec = context.system.dispatcher
   implicit val logging = new AkkaLogging(context.system.log)
   var rescheduleJob = false // true iff actor receives a job but cannot process it because actor will destroy itself
+  var activeActivationCount = 0
```

Comment:
WDYT about making this part of `WarmedData` or even `ContainerData`? That would eschew the need for the `ActivationCounter` and make other parts easier as well (I'll ad comments for posterity)